sites/app/modules/app/client/mixins/mixins.Likeable.jsx

```diff
@@ -15,6 +15,7 @@ const Likeable = {
 
     if (!Meteor.userId()) {
       this.props.dispatch(modal.render(OnBoard, { coverHeader: true }))
+      this.props.dispatch(modal.update({modalBackground: "light"}))
```

Comment:
This is what was causing the header to still be displayed when hitting "like" and triggering the sign up form. We need to combined these dispatches to:

```javascript
this.props.dispatch(modal.render(Onboard, {
  coverHeader: true,
  modalBackground: "light",
});
```